content/docs/user-guide/components/reference/atom/debug-rendering.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Debug Render Component
+linktitle: Debug Render Component
+description: 'Open 3D Engine (O3DE) debug rendering level component reference.'
+toc: true
+---
+
+**Note**: This is a level component, and as such must be added to the level instead of an entity.
+
+{{< image-width "/images/user-guide/components/reference/atom/debug-rendering/debug-rendering-component.jpg" "500" "Debug Render Component" >}}
+
+
+## Debug Visualization Properties
+
+The **Debug Rendering Level Component** is used to visualize rendering information about the scene, such as material properties like albedo and roughness or lighting factors like direct/indirect and diffuse/specular.
+
+| Property | Description |
+|-|-|
+| **Enable Render Debugging** | Whether to utilize render debugging or not. If set to false, properties in this component have no effect. |
+| **Debug View Mode** | Specifies what debug information is displayed in the viewport (for example material albedo, metalness, roughness...). Note that the options "Albedo" and "BaseColor" differ in that BaseColor will display the color used for metal reflectance, whereas Albedo for metals will be zero/black. |
+| **Lighting Type** | Whether to display Diffuse Lighting, Specular Lighting, or both. |
+| **Lighting Source** | Whether to display Direct Lighting, Indirect Lighting, both, or use a debug directional light as the only light source (see the next section). |
+
+Screeshot with **Debug View Mode = Albedo**
+
+{{< image-width "/images/user-guide/components/reference/atom/debug-rendering/albedo.jpg" "1500" "Screeshot with Debug View Mode set to Albedo" >}}
+
+
+Screenshot with **Lighting Type = Diffuse** and **Lighting Source = Indirect** (i.e. the scene will only apply indirect diffuse lighting)
```

Comment:
Move to **Examples** section at the bottom of the page (per earlier comment on reorganizing).
```suggestion
Example of debug lighting, with **Lighting Type** set to `Diffuse` and **Lighting Source** set to `Indirect`:
```